AmplifyPlugins/API/AWSAPICategoryPluginIntegrationTests/GraphQL/GraphQLWithUserPoolIntegrationTests/GraphQLWithUserPoolIntegrationTests.swift

```diff
@@ -7,6 +7,7 @@
 
 import XCTest
 @testable import Amplify
+import AmplifyPlugins
 import AWSMobileClient
```

Comment:
can we remove this as well?

Reply:
Yes we can remove that. We'll only need `AWSCore` to call `AWSInfo.configureDefaultAWSInfo` on line 41

Reply:
i think `AWSInfo.configureDefaultAWSInfo` uses SDk as well, though it may be working, we should be configuring the AWSCognitoAuthPlugin if we are removing the `import AWSMobileClient`